app/src/main/java/org/schabi/newpipe/views/AnimatedProgressBar.java

```diff
@@ -26,6 +30,23 @@ public AnimatedProgressBar(final Context context, final AttributeSet attrs,
         super(context, attrs, defStyleAttr);
     }
 
+    /**
+     * For video items in lists, progress bar for playback information should only display when
+     * option `Positions in lists` is enabled.
+     * @return whether `Positions in lists` is enabled
+     */
+    public Boolean getPositionsInListsEnabled() {
+        final Context context = getContext();
+        final SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
+
+        return prefs.getBoolean(context.getString(
+                        R.string.enable_watch_history_key), true)
+                && prefs.getBoolean(context.getString(
+                        R.string.enable_playback_resume_key), true)
+                && prefs.getBoolean(context.getString(
+                        R.string.enable_playback_state_lists_key), true);
+    }
+
```

Comment:
Please remove this from the ProgressBar.
A progressbar is unrelated to the settings...

Reply:
The settings are needed to fix #4053, to solve this issue, option `Positions in lists` should control the display of progress bar in video lists